crates/rpc/rpc-builder/Cargo.toml

```diff
@@ -66,3 +66,6 @@ reth-node-api.workspace = true
 tokio = { workspace = true, features = ["rt", "rt-multi-thread"] }
 serde_json.workspace = true
 clap = { workspace = true, features = ["derive"] }
+
+[features]
+reth-rpc-js-tracer = ["reth-rpc/js-tracer"]
```

Comment:
can we just call this `js-tracer`

Reply:
Yes, done in 3e5f678f8431096f36ec7185de038b7a72b2b0c4